osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -64,7 +84,7 @@ protected override double StrainValueOf(DifficultyHitObject current)
                 }
             }
 
-            return (1 + (speedBonus - 1) * 0.75) * angleBonus * (0.95 + speedBonus * Math.Pow(distance / single_spacing_threshold, 3.5)) / osuCurrent.StrainTime;
+            return (1 + (speedBonus - 1) * 0.75) * angleBonus * (0.95 + speedBonus * Math.Pow(distance / single_spacing_threshold, 3.5)) / deltaTime;
```

Comment:
Same as in aim, it is possible to get a div by 0 here. I think you can do max(deltaTime, 20) or something simple.